### PR TITLE
Update Rust smaple

### PR DESCRIPTION
**Pull request by Sevenannn**

### CODE REVIEW

File: rust/snippets/spice-rs/main.rs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 use futures::stream::StreamExt;
 use spice_rs::*;
 
-let mut spice_client = new_spice_client(api_key).await.expect("Failed to new spice client");
+let mut spice_client = new_spice_client(api_key).await;
 
 match spice_client.query(
-    "SELECT number, \"timestamp\", base_fee_per_gas, base_fee_per_gas / 1e9 AS base_fee_per_gas_gwei FROM eth.recent_blocks limit 10".to_string(),
-    None).await {
+    "SELECT number, \"timestamp\", base_fee_per_gas, base_fee_per_gas / 1e9 AS base_fee_per_gas_gwei FROM eth.recent_blocks limit 10".to_string()
```
Comment (mitchdevenport) [inline rust/snippets/spice-rs/main.rs:7]: string literals aren't strings in rust?

Comment (Sevenannn) [reply rust/snippets/spice-rs/main.rs:7]: Only "" are strings in rs, so \ are needed for escaping "" around timestamp

Comment (mitchdevenport) [reply rust/snippets/spice-rs/main.rs:7]: I'm referring to calling `to_string()` on a string literal. Why does the string literal need to be converted to a string?

**Review by mitchdevenport — APPROVED** [review]

File: rust/snippets/spice-rs/main.rs (hunk 1 above)
Comment (Sevenannn) [reply rust/snippets/spice-rs/main.rs:7]: &str is the "" in Rust

**Pull request merged**